web/src/admin/users/UserDevicesList.ts

```diff
@@ -36,41 +36,30 @@ export class UserDeviceList extends MFADevicesPage {
             });
     }
 
+    columns(): TableColumn[] {
+        const headers = [msg("Name"), msg("Type"), msg("Confirmed")];
+        return headers.map((th) => new TableColumn(th, ""));
```

Comment:
I think this is more readable with just returning an array of TableColumn instances imo (a very subjective nit)